src/dmd/astbase.d

```diff
@@ -5419,25 +5425,6 @@ struct ASTBase
         }
     }
 
-    extern (C++) final class ThrowExp : UnaExp
```

Comment:
?

Reply:
I modified the PR to not include this change